### PR TITLE
Made history longer

### PR DESCRIPTION
**Pull request by Wuerfel21**
It was too short, i always needed to retype long commands

### CODE REVIEW

File: src/main/resources/assets/opencomputers/loot/OpenOS/bin/sh.lua [hunk 1]
```diff
@@ -165,7 +165,7 @@ if #args == 0 and (io.input() == io.stdin or options.i) and not options.c then
         term.write("exit\n")
         return -- eof
       end
-      while #history > 10 do
+      while #history > 20 do
```
Comment (Wuerfel21) [inline src/main/resources/assets/opencomputers/loot/OpenOS/bin/sh.lua]: derp, 2 times more history

**Comment by fnuecke** [comment]
That's still kinda arbitrary, and the low limit was intended to avoid (relatively) non-obvious memory consumption. I think the better approach would be to add a `HISTSIZE` env var (defaulting to 10) and use that there. Then you can just `set HISTSIZE=20` in the shell and be happy ;-)

**Comment by Wuerfel21** [comment]
Please hold the line, your request is beeing processed...

**Comment by Wuerfel21** [comment]
Well, I done it(hopefully right)

File: src/main/resources/assets/opencomputers/loot/OpenOS/bin/sh.lua [hunk 2]
```diff
@@ -7,6 +7,14 @@ local shell = require("shell")
 local term = require("term")
 local text = require("text")
 
+local function getHistSize()
```
Comment (Wuerfel21) [inline src/main/resources/assets/opencomputers/loot/OpenOS/bin/sh.lua]: This Function does stuff(really?) to either return HISTSIZE or 10

File: src/main/resources/assets/opencomputers/loot/OpenOS/boot/01_os.lua [hunk 3]
```diff
@@ -16,6 +16,7 @@ local env = {
   SHELL="/bin/sh",
   TMP="/tmp", -- Depricated
   TMPDIR="/tmp"
```
Comment (fnuecke) [inline src/main/resources/assets/opencomputers/loot/OpenOS/boot/01_os.lua]: Missing a comma there. Test your stuff! :O

File: src/main/resources/assets/opencomputers/loot/OpenOS/bin/sh.lua [hunk 4]
```diff
@@ -165,7 +173,7 @@ if #args == 0 and (io.input() == io.stdin or options.i) and not options.c then
         term.write("exit\n")
         return -- eof
       end
-      while #history > 10 do
+      while #history > getHistSize() do
```
Comment (fnuecke) [inline src/main/resources/assets/opencomputers/loot/OpenOS/bin/sh.lua]: Maybe just a `(tonumber(os.getenv("HISTSIZE")) or 10)` here, do away with the function?

File: src/main/resources/assets/opencomputers/loot/OpenOS/boot/01_os.lua (hunk 3 above)
Comment (fnuecke) [inline src/main/resources/assets/opencomputers/loot/OpenOS/boot/01_os.lua]: Also, probably better make it `HISTSIZE="10"`, for consistency (don't mix in non-string types into env-vars).

**Comment by Wuerfel21** [comment]
ok

**Comment by fnuecke** [comment]
Thanks :)

**Comment by Wuerfel21** [comment]
Phew... No more typeing madness...

**Pull request merged**